next.config.js

```diff
@@ -2,6 +2,18 @@
 const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
+  async rewrites() {
+    return [
+      {
+        source: '/api/:path*',
+        destination: 'http://localhost:3000/api/:path*'
```

Comment:
As stated in a comment of the previous PR, I think it's better if the base API endpoint is provided from a .env variable, so it's easier to change that if is hard coded

Reply:
done, thanks!